packages/web-components/src/components/va-card/va-card.tsx

```diff
@@ -0,0 +1,41 @@
+import { 
+  Component,
+  Host, 
+  Prop, 
+  h 
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Card
+ * @maturityCategory dont_use
+ * @maturityLevel proposed
+ */
+
+@Component({
+  tag: 'va-card',
+  styleUrl: 'va-card.scss',
+  shadow: true,
+})
+export class VaCard {
+  /**
+   * If `true`, a drop-shadow will be displayed
+   */
+  @Prop() showShadow?: boolean = false;
+
+  render() {
+    const {
+      showShadow
+    } = this;
+
+    const classes = classnames('va-card',  {'show-shadow': showShadow});
+
+    return (
+      <Host>
+        <div class={classes}>
```

Comment:
Can you leverage the `<Host>` element for applying the classes? In the dom, this would just be the `<va-card>` element so you should be able to add style directly to it (that would save you from having the extra div).

In CSS, style is applied to the web component top level element like:
 
`:host { }`

Example: https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-omb-info/va-omb-info.css#L1-L3